examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -0,0 +1,1191 @@
+ax.set_xlabel("Alpha")
+ax.set_title(
+    "Posterior Predictive Expected Failure Count between 150-600 hours \nas a function of Weibull(alpha, beta)",
+    fontsize=20,
+)
+
+ax1.hist(
+    output_df["lb"],
+    ec="black",
+    color="slateblue",
+    label="95% PI Lower Bound on Failure Count",
+    alpha=0.3,
+)
+ax1.axvline(output_df["lb"].mean(), label="Expected 95% PI Lower Bound on Failure Count")
+ax1.axvline(output_df["ub"].mean(), label="Expected 95% PI Upper Bound on Failure Count")
+ax1.hist(
+    output_df["ub"],
+    ec="black",
+    color="cyan",
+    label="95% PI Upper Bound on Failure Count",
+    bins=20,
+    alpha=0.3,
+)
+ax1.hist(
+    output_df["expected"], ec="black", color="pink", label="Expected Count of Failures", bins=20
+)
+ax1.set_title("Uncertainty in the Posterior Prediction Interval of Failure Counts", fontsize=20)
+ax1.legend()
+
+ax2.set_title("Expected Costs Distribution(s)  \nbased on implied Failure counts", fontsize=20)
+ax2.hist(
+    cost_func(output_df["expected"], 2.3),
+    label="Cost(failures,2)",
+    color="royalblue",
+    alpha=0.3,
+    ec="black",
+    bins=20,
+)
+ax2.hist(
+    cost_func(output_df["expected"], 2),
+    label="Cost(failures,2.3)",
+    color="red",
+    alpha=0.5,
+    ec="black",
+    bins=20,
+)
+ax2.set_xlabel("$ cost")
+ax2.legend()
+```
+
+The choice of model in such cases is crucial. The decision about which failure profile is apt has to be informed by a subject matter expert because extrapolation from such sparse data is always risky. An understanding of the uncertainty is crucial if real costs attach to the failures and the subject matter expert is usually better placed to tell if you 2 or 7 failures can be plausibly expected within 600 hours of service. 
+
+# Conclusion
+
+We've seen how to analyse and model reliability from both a frequentist and Bayesian perspective and compare against the non-parametric estimates. We've shown how prediction intervals can be derived for a number of key statistics by both a bootstrapping and a bayesian approach. We've seen approaches to calibrating these prediction intervals through re-sampling methods and informative prior specification. These views on the problem are complementary and the choice of technique which is appropriate should be driven by factors of the questions of interest, not some ideological commitment. 
+
+In particular we've seen how the MLE fits to our bearings data provide a decent first guess approach to establishing priors in the Bayesian analysis. We've also seen how subject matter expertise can be elicited by deriving key quantities from the implied models and subjecting these implications to scrutiny. The choice of Bayesian prediction interval is calibrated to our priors expectations, and where we have none - we can supply vague or non-informative priors. The implications of these priors can again be checked and analysed against an appropriate cost function.
+
++++
+
+## Authors
+
+Nathaniel Forde
```

Comment:
This should follow the same format as other notebooks: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#authorship-and-attribution

Reply:
Updated to better reflect the format.